src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/InputWithLabel.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {FC, useCallback, useEffect, useState} from 'react'
+import {
+  AlignItems,
+  ComponentSize,
+  FlexBox,
+  FlexDirection,
+  Form,
+  FormElementProps,
+  Input,
+  InputLabel,
+  InputProps,
+  JustifyContent,
+} from '@influxdata/clockface'
+
+// Style
+import './InputWithLabel.scss'
+
+interface OwnProps {
+  description?: string
+  errorMessage?: string
+  showError?: boolean
+}
+
+type Props = OwnProps & FormElementProps & InputProps
+
+const DEFAULT_ERROR_MESSAGE = 'This is a required field'
+
+export const InputWithLabel: FC<Props> = ({
+  label,
+  required,
+  onChange,
+  description,
+  value,
+  errorMessage = DEFAULT_ERROR_MESSAGE,
+  showError = false,
+  ...args
+}) => {
+  const [isDirty, setIsDirty] = useState(false)
+  const [inputVal, setInputVal] = useState(value)
+  const [hasError, setHasError] = useState(showError)
+
+  const handleChange = useCallback(
+    e => {
+      const val = e.target.value
+      if (val !== inputVal) {
```

Comment:
I would change val to something more descriptive so that we know what `val` versus `inputVal` are.

Reply:
Changed to `newValue`. Feel free to suggest a better name.